src/redash_stmo/integrations/iodide/extension.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+import requests
+import json
+import logging
+import os
+
+from redash.models import Query
+from redash.handlers.base import BaseResource, get_object_or_404
+from redash.permissions import require_permission
+
+from redash_stmo import settings
+from redash_stmo.resources import add_resource
+
+
+logger = logging.getLogger(__name__)
+
+
+class IodidePostResource(BaseResource):
```

Comment:
This should be called `IodideNotebookResource` since it's what it handles, not just POST requests.